test/cbmc/proofs/SigV4_GenerateHTTPAuthorization/Makefile

```diff
@@ -0,0 +1,27 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+HARNESS_ENTRY = harness
+HARNESS_FILE = SigV4_GenerateHTTPAuthorization_harness
+
+# This should be a unique identifier for this proof, and will appear on the
+# Litani dashboard. It can be human-readable and contain spaces if you wish.
+PROOF_UID = SigV4_GenerateHTTPAuthorization
+
+DEFINES +=
+INCLUDES +=
+
+
+REMOVE_FUNCTION_BODY +=
+UNWINDSET +=
+
+PROOF_SOURCES += $(PROOFDIR)/$(HARNESS_FILE).c
+PROOF_SOURCES += $(SRCDIR)/test/cbmc/stubs/sigv4_stubs.c
```

Comment:
If you have not stubbed any function for this proof. There is no need to include this file in the proof.